packages/schematics/angular/service-worker/files/ngsw-config.json

```diff
@@ -1,4 +1,5 @@
 {
+  "$schema": "./node_modules/@schematics/angular/service-worker/config/schema.json",
```

Comment:
When having multiple apps inside the workspace the `ngsw-config.json` is not generated in the root of the workspace, but in a sub directory example: `/projects/myapp/src/ngsw-config.json`, this will cause the schema not to be found.

You'd need to do something like: https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/application/files/root/tsconfig.app.json#L2

Reply:
Is the relativePathToWorkspaceRoot variable available in this context too?

Reply:
No, though you can add it. See: https://github.com/angular/angular-cli/blob/2326850374cd393a825e339beb5846afcff87f8a/packages/schematics/angular/application/index.ts#L284-L295

Reply:
OK. I won't be confident to do this part without testing, but as it happens at generation, I don't know how to test. How can I create an Angular project from the Angular CLI from my fork repo?

Reply:
https://github.com/angular/angular-cli/blob/master/packages/angular/cli/README.md#development-hints-for-working-on-angular-cli

Though in your case you'd need to link the `@schematics/angular` package.

Or even better create a Unit Test.

Reply:
I'll manage this part when angular/angular#27859 is merged.